pom.xml

```diff
@@ -1791,11 +1708,6 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
           <!-- @argLine is filled by jacoco maven plugin. @{} means late evaluation -->
           <argLine>${maven-surefire-plugin.argLine} @{argLine}</argLine>
           <environmentVariables>
-            <HADOOP_COMMON_HOME>${hadoop.common.build.dir}</HADOOP_COMMON_HOME>
-            <!-- HADOOP_HOME required for tests on Windows to find winutils -->
-            <HADOOP_HOME>${hadoop.common.build.dir}</HADOOP_HOME>
-            <LD_LIBRARY_PATH>${env.LD_LIBRARY_PATH}:${project.build.directory}/native/target/usr/local/lib:${hadoop.common.build.dir}/native/target/usr/local/lib</LD_LIBRARY_PATH>
```

Comment:
I don't know if we need to remove the native library in the tests. It's probably okay.
Other than this one I am +1.

Reply:
Thanks for the review @jojochuang.  This seems to have been copied from `hadoop-project/pom.xml`.  I don't think Ozone has any native library.

Reply:
yeah. I think the assumption here is the behavior is the same with the native Hadoop library or not, which makes sense to me.

Reply:
The only native library I remembered for ozone is via ratis-thirdparty jar. So I'm OK with removing it.